src/Model/Behavior/ExposeBehavior.php

```diff
@@ -82,7 +82,7 @@ public function findExposedList(Query $query, array $options) {
 	 * @return string
 	 */
 	public function getExposedKey(): string {
```

Comment:
Should we make this an option?

    $prefixed = false

Then people can use it if needed only.